package.json

```diff
@@ -26,8 +27,11 @@
     "!(*.js)": "prettier --write --ignore-unknown"
   },
   "devDependencies": {
+    "@eslint/js": "^9.0.0",
```

Comment:
This doesn't seem necessary?

Reply:
I can see that at lot of places there are mentions of the same package like. But I wonder how this appeared out of no where. I never installed it 🤔 

https://github.com/eslint/rewrite/blob/19a755b85e4d1f9299b32293e12221406b0c1abc/packages/migrate-config/tests/fixtures/reveal-md/expected.mjs#L7

Reply:
It was removed in https://github.com/eslint/rewrite/pull/43/files 🤔 how is it getting referenced? I don't see it as internal package as well?

Reply:
```suggestion
```
feel free to commit this and merge in case its not required